apis/apps/v1alpha1/apicast_types.go

```diff
@@ -28,6 +30,21 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// CustomPolicySpec contains or has reference to an APIcast custom policy
+type CustomPolicySpec struct {
+	// Name specifies the name of the custom policy
+	Name string `json:"name"`
+	// Version specifies the name of the custom policy
+	Version string `json:"version"`
+
+	// SecretName specifies the secret name holding the custom policy metadata and lua code
+	SecretName string `json:"secretName"`
```

Comment:
If we are going to reference a secret and always in the same namespace, can we use v1.LocalObjectReference as with other fields like HTTPSCertificateSecretRef and AdminPortalCredentialsRef?

Reply:
updated

Reply:
:+1: